docker/archimedes-base/entrypoints/build.sh

```diff
@@ -24,7 +24,6 @@ if ! [ -e config.yml ] && [ -e config.yml.template ]; then
   cp config.yml.template config.yml
 fi
 
-poetry completions bash > /app/poetry.completion
```

Comment:
This was what caused the excessively archimedes rebuilding, sadly.  This ran on every build, which updated the source file, which required a new rebuild everytime.

Reply:
Does that mean no poetry autocompletions in the Archimedes container now, or that is still available if you do `make -C archimedes bash`?

Reply:
No, it still works.  This actually exists in the source directory anyways thanks to the mount point in docker-compose dropping it.  So it gets checked in.  what it does mean is that if we want updated autocompletions, we just have to rerun it inside the docker-compose bash shell and commit it.